plugins/inventory/aws_ec2.py

```diff
@@ -537,14 +567,35 @@ def _get_hostname(self, instance, hostnames):
             else:
                 return to_text(hostname)
 
-    def _query(self, regions, filters, strict_permissions):
+    def _query(self, regions, include_filters, exclude_filters, strict_permissions):
         '''
             :param regions: a list of regions to query
-            :param filters: a list of boto3 filter dictionaries
+            :param include_filters: a list of boto3 filter dictionaries
+            :param exclude_filters: a list of boto3 filter dictionaries
             :param hostnames: a list of hostname destination variables in order of preference
+            :param includes_entries_matching: a list of filters, instances matching these filters are included in the result
```

Comment:
```suggestion
```

hostnames and includes_entries_matching look to be gone?

Reply:
`include_filters` and `exclude_filters` supersede` includes_entries_matching`. For `hostnames` it's a mistake.